src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/MetricItemExtensions.cs

```diff
@@ -136,173 +133,173 @@ internal static OtlpMetrics.Metric ToOtlpMetric(this IMetric metric)
-                        : OtlpMetrics.AggregationTemporality.Cumulative,
-                        };
-
-                        var dataPoint = new OtlpMetrics.HistogramDataPoint
+                        foreach (var metricPoint in metric.GetMetricPoints())
                         {
-                            StartTimeUnixNano = (ulong)metric.StartTimeExclusive.ToUnixTimeNanoseconds(),
-                            TimeUnixNano = (ulong)metric.EndTimeInclusive.ToUnixTimeNanoseconds(),
-                            Count = (ulong)histogramMetric.PopulationCount,
-                            Sum = histogramMetric.PopulationSum,
-                        };
-
-                        foreach (var bucket in histogramMetric.Buckets)
-                        {
-                            dataPoint.BucketCounts.Add((ulong)bucket.Count);
+                            var dataPoint = new OtlpMetrics.NumberDataPoint
+                            {
+                                StartTimeUnixNano = (ulong)metricPoint.StartTime.ToUnixTimeNanoseconds(),
+                                TimeUnixNano = (ulong)metricPoint.EndTime.ToUnixTimeNanoseconds(),
+                            };
 
-                            // TODO: Verify how to handle the bounds. We've modeled things with
-                            // a LowBoundary and HighBoundary. OTLP data model has modeled this
-                            // differently: https://github.com/open-telemetry/opentelemetry-proto/blob/bacfe08d84e21fb2a779e302d12e8dfeb67e7b86/opentelemetry/proto/metrics/v1/metrics.proto#L554-L568
-                            dataPoint.ExplicitBounds.Add(bucket.HighBoundary);
-                        }
+                            for (int i = 0; i < ((metricPoint.Keys == null) ? 0 : metricPoint.Keys.Length); i++)
+                            {
+                                KeyValuePair<string, object> tag = new KeyValuePair<string, object>(metricPoint.Keys[i], metricPoint.Values[i]);
+                                dataPoint.Attributes.Add(tag.ToOtlpAttribute());
+                            }
 
-                        // TODO: Do TagEnumerationState thing.
-                        foreach (var attribute in metric.Attributes)
-                        {
-                            dataPoint.Attributes.Add(attribute.ToOtlpAttribute());
+                            dataPoint.AsDouble = metricPoint.DoubleValue;
+                            gauge.DataPoints.Add(dataPoint);
                         }
 
-                        foreach (var exemplar in histogramMetric.Exemplars)
-                        {
-                            dataPoint.Exemplars.Add(exemplar.ToOtlpExemplar());
-                        }
-
-                        otlpMetric.Histogram = histogram;
+                        otlpMetric.Gauge = gauge;
                         break;
                     }
 
-                case MetricType.Summary:
```

Comment:
I assume you intended to remove `Summary` for now since the aggregation for summary metrics is not defined in the spec?

Reply:
yes will remove! thanks for reminder